build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'java'
+    id 'org.jetbrains.intellij' version '0.4.8'
+}
+
+group 'com.slacklane'
```

Comment:
Should be com.stacklane "t" not "l"

Reply:
I thought that I had committed that to memory because initially I was really messing that up :)